frontend/javascripts/admin/tasktype/task_type_create_view.tsx

```diff
@@ -43,464 +62,448 @@ function isValidMagnification(_rule, value) {
   }
 }
 
-// @ts-expect-error ts-migrate(7006) FIXME: Parameter 'rawSettings' implicitly has an 'any' ty... Remove this comment to see the full error message
-function getMagnificationAdaptedSettings(rawSettings) {
-  const { resolutionRestrictionsForm, ...settingsWithoutMagnifications } = rawSettings;
-  const resolutionRestrictions = {
-    min: resolutionRestrictionsForm.shouldRestrict ? resolutionRestrictionsForm.min : null,
-    max: resolutionRestrictionsForm.shouldRestrict ? resolutionRestrictionsForm.max : null,
-  };
-
-  if (
-    resolutionRestrictions.min != null &&
-    resolutionRestrictions.max != null &&
-    resolutionRestrictions.min > resolutionRestrictions.max
-  ) {
-    Toast.error("Minimum resolution must not be greater than maximum resolution.");
-    return null;
+function isMinimumMagnifactionLargerThenMaxRule(value: number | undefined, maxMag: number) {
```

Comment:
Add two new validation rules to check that min restricted resolution is smaller then max and vice versa. In the old code, we only checked this after a successful form submit. In this version, the form checks this while the user inputs values and give direct validation feedback.